README.md

```diff
@@ -80,6 +80,22 @@ publisher, _ := pubsub.NewPublisher(
 )
 ```
 
+### Initializing a publisher with duplication detection
+Duplication detection cannot be enabled on topics that already exist.
```

Comment:
Should we always enable duplicate detection, by default?
what's the downside?

Reply:
ah, can you change the dedup window after topic creation?

Reply:
unfortunately you can't change the dedupe window after topic creation (SB-side limitation). we discussed offline and decided that we will allow for using duplicate detection but will park the decision of whether to enable it by default.